srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -27,23 +183,34 @@
     // if not.
     private RiskModel importedModel = null;
     
-    protected EditRiskModel(final RiskModel rm)
+    protected EditRiskModel(final RiskModel rm, final List<Specialty> applSpecialty)
     {
         fRiskModel = rm;
         modelName = rm.getDisplayName();
         
+        applicableSpecialties = ImmutableList.copyOf(applSpecialty);
         importedModel = null;
```

Comment:
See above. Also, importedModel is placed after the inner class, but I think the inner class should be at the bottom of the class, therefore making the class more readable and making all private variable declarations for EditRiskModel appear next to each other.